src/composables/Alchemy.ts

```diff
@@ -13,6 +17,20 @@ export type Vial = {
   effect: string;
 };
 
+export type Bubble = {
+  Name: string;
+  x1: string;
+  x2: string;
+  Func: string;
```

Comment:
Ooh I remember seeing this before? I had asked someone to change this to be a [function type](https://www.typescriptlang.org/docs/handbook/functions.html) and not have separate variables for `x1` and `x2`.

Reply:
That person was me, but even after looking at that link I am not sure what I am meant to do